native/src/main/java/io/ballerina/lib/redis/connection/RedisConnectionManager.java

```diff
@@ -361,13 +353,41 @@ private BaseRedisCommands<K, V> getCommandConnection() throws RedisConnectorExce
         }
     }
 
-    /**
-     * Represents a Redis server address.
-     *
-     * @param host redis server host
-     * @param port redis server port
-     */
-    private record ServerAddress(String host, int port) {
+    private SslOptions constructSslOptions(SecureSocket secureSocket) {
+
+        SslOptions.Builder sslOptionsBuilder = SslOptions.builder();
+
+        TrustStore trustStore = secureSocket.trustStoreCert();
+        if (trustStore != null) {
```

Comment:
```suggestion
        if (Objects.nonNull(trustStore)) {
```

Reply:
Using the utility methods in the `Objects` class is as per the standard java coding guidelines. I would prefer this to be a practice in all of our library modules with java code.

Reply:
Yes agreed. What is mentioned was, as this convention has been used our redis code base, we'll need to convert all the null checks to this to make it consistent if we do so. As that'll be a deviation from the purpose of this PR, we can work on that in a separate PR isnt it?